tests/e2e/cucumber/steps/ui/public.ts

```diff
@@ -114,7 +114,7 @@ When(
       await pageObject.upload({
         to: info.to,
         resources: [this.filesEnvironment.getFile({ name: info.resource })],
-        createVersion: info.create_version === 'true'
+        option: info.option !== undefined ? info.option : null
```

Comment:
```suggestion
        option: info.option
```

Reply:
if nothing pass to info.option then the option data type will be **undefined** and `undefined data type will be false `. So we can use undefined data type directly